app/Services/NumberFormatter.php

```diff
@@ -176,7 +176,7 @@ public static function hasSymbol(string $currency): bool
         return config('currencies.currencies.'.strtolower($currency).'.symbol') !== null;
     }
 
-    public static function gweiToArk(float $value, bool $withSuffix = true): string
+    public static function gweiToArk(string | int | float $value, bool $withSuffix = true): string
```

Comment:
method was added in https://github.com/ArdentHQ/arkscan/pull/1025, but maybe we should move this to php-crypto instead? probably useful in general to have a method like that 🤔

Reply:
https://app.clickup.com/t/86dvb6env added a card for that :oleglala: